shared/datastore.go

```diff
@@ -25,6 +25,23 @@ func LoadTestRun(ctx context.Context, id int64) (*TestRun, error) {
 	return &testRun, nil
 }
 
+// LoadTestRunsBySHAs loads all test runs that belong to any of the given revisions (SHAs).
+func LoadTestRunsBySHAs(ctx context.Context, shas ...string) (runs TestRuns, err error) {
+	for _, sha := range shas {
+		if len(sha) > 10 {
+			sha = sha[:10]
```

Comment:
Aren't we storing the full sha in datastore now? Can we avoid the shortening? (Not because ambiguity is a high risk, but because it'd be less code.)

Reply:
We are, but it's not indexed. See #49

Reply:
Ah, thanks.